### PR TITLE
Extension of the language setting option

### PR DESCRIPTION
**Pull request by AxelHamburch**
With this PR comes the option to set the language for the displays in the config.ini. The list of possible languages ​​was also expanded at the same time.
  
Since the language texts are of different lengths and have to be individually adjusted by the user on each display, I created a documentation page and added it to the guideline under Options.

### CODE REVIEW

File: docs/guide/we_need_your_help.md [hunk 1]
```diff
@@ -25,19 +17,20 @@ Then we add the following text between `centname = cent` and `# Set the Fee in %
     # Code 2 from https://www.science.co.il/language/Codes.php
     language = en
 
-    # Do you have a camera? "no" or "yes"
-    camera = no
+    # Do you have a camera? "False" or "True"
```
Comment (lightisfaster) [inline docs/guide/we_need_your_help.md:20]: Great, I hadn't seen it

**Review by lightisfaster — COMMENTED** [review]
Thanks for this PR: A small change should be made, see review -> if, elif, else

File: docs/guide/display.md [hunk 2]
```diff
@@ -20,6 +20,8 @@ $ sudo python3 ./epd_2in13_V2_test.py
 
 If everything has been correctly connected and installed, the display will now show a demonstration and finally the screen will cleare.
 
+`Help:`If it dosn't work, try the D-Version (flexible): `$ sudo python3 ./epd_2in13d_test.py` or one of [this](https://github.com/waveshare/e-Paper/tree/master/RaspberryPi_JetsonNano/python/examples).
```
[the remark this answers is not in the document]
Comment (AxelHamburch) [reply docs/guide/display.md:23]: Do the look different? My v2 looks like the d-version. It is just not flexible.

File: displays/inkyphat.py [hunk 3]
```diff
@@ -6,10 +6,33 @@
...
+from displays import messages_fr
+from displays import messages_it
+from displays import messages_pt
+from displays import messages_tr
+
+
+# Select language from config.ini. If no match => English
+if config.conf["atm"]["language"] == "de":
+    messages = messages_de
+if config.conf["atm"]["language"] == "es":
```
[the remark this answers is not in the document]
Comment (AxelHamburch) [reply displays/inkyphat.py]: done

File: displays/papiruszero2in.py [hunk 4]
```diff
@@ -6,10 +6,33 @@
...
+from displays import messages_fr
+from displays import messages_it
+from displays import messages_pt
+from displays import messages_tr
+
+
+# Select language from config.ini. If no match => English
+if config.conf["atm"]["language"] == "de":
+    messages = messages_de
+if config.conf["atm"]["language"] == "es":
```
[the remark this answers is not in the document]
Comment (AxelHamburch) [reply displays/papiruszero2in.py]: done

File: displays/waveshare2in13v2.py [hunk 5]
```diff
@@ -6,10 +6,33 @@
...
+from displays import messages_fr
+from displays import messages_it
+from displays import messages_pt
+from displays import messages_tr
+
+
+# Select language from config.ini. If no match => English
+if config.conf["atm"]["language"] == "de":
+    messages = messages_de
+if config.conf["atm"]["language"] == "es":
```
[the remark this answers is not in the document]
Comment (AxelHamburch) [reply displays/waveshare2in13v2.py]: done

File: displays/waveshare2in66.py [hunk 6]
```diff
@@ -6,10 +6,33 @@
...
+from displays import messages_fr
+from displays import messages_it
+from displays import messages_pt
+from displays import messages_tr
+
+
+# Select language from config.ini. If no match => English
+if config.conf["atm"]["language"] == "de":
+    messages = messages_de
+if config.conf["atm"]["language"] == "es":
```
[the remark this answers is not in the document]
Comment (AxelHamburch) [reply displays/waveshare2in66.py]: done

**Comment by AxelHamburch** [comment]
Done all changes!

**Pull request merged**